safedata_validator/entry_points.py

```diff
@@ -82,6 +82,9 @@ def _safedata_validator_cli(args_list: Optional[list[str]] = None) -> None:
             providing a list of command line argument strings to the entry point
             function. For example, ``safedata_validate show_resources`` can be
             replicated by calling ``_safedata_validate_cli(['show_resources'])``.
+
+    Returns:
+        An integer code showing success (0) or failure (1).
```

Comment:
Is an integer used here because it's extensible (e.g. 2 could be returned to indicate something), because otherwise couldn't this be a bool?

Reply:
I don't know if it makes a difference here, because I think the shell would interpret bool as 1 or 0, but shell exit codes are canonically integers that can take a range of values (e.g. https://tldp.org/LDP/abs/html/exitcodes.html)